composer

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+############################################
+# PHP Composer inside php docker container #
+############################################
+
+# Passing arguments from this script to docker exec command
+
+docker-compose exec php composer "$@"
```

Comment:
Thoughts on moving this file into a `/bin` directory? 

This would follow the composer recommendation and we can add `./bin` to `$PATH` for all projects using this convention.

Reply:
You mean into webtechdeploy?

Reply:
No, an actual directory called `bin`. I'll PR after we finally get this sprint done :)